spidermon/utils/web.py

```diff
@@ -0,0 +1,23 @@
+import sys
+
+if sys.version_info[0] == 3:
+    from urllib.parse import urlparse
+    from urllib.request import urlopen
+else:
+    from urlparse import urlparse
+    from urllib import urlopen
+
+
+def is_schema_url(path):
+    result = urlparse(path)
+    try:
+        if all([result.scheme, result.netloc, result.path]):
+            return True
+        return False
+    except AttributeError:
```

Comment:
The only call that could raise `AttributeError` is `urlparse` that is outside the `try` block.

Reply:
Not really, if schema doesn't have scheme result.scheme will raise an attribute error.